UPGRADE-5.2.md

```diff
@@ -16,6 +16,28 @@ FrameworkBundle
    used to be added by default to the seed, which is not the case anymore. This allows sharing caches between
    apps or different environments.
 
+Form
+----
+
+ * Deprecated `PropertyPathMapper` in favor of `DataMapper` and `PropertyPathAccessor`.
+
+ Before:
+
+ ```php
+ use Symfony\Component\Form\Extension\Core\DataMapper\PropertyPathMapper;
+
+ $builder->setDataMapper(new PropertyPathMapper());
+ ```
+
+ After:
+
+ ```php
+ use Symfony\Component\Form\Extension\Core\DataAccessor\PropertyPathAccessor;
+ use Symfony\Component\Form\Extension\Core\DataMapper\DataMapper;
+
+ $builder->setDataMapper(new DataMapper(new PropertyPathAccessor()));
```

Comment:
could be just `$builder->setDataMapper(new DataMapper());`, isn't it? since the default is to use a `PropertyPathAccessor` when no argumetn is provided

Reply:
You're right, technically it is not necessary, but visually it is consistent with the depreciation message and self-explanatory, otherwise we would have to explain it.